### PR TITLE
Add dedicated encryption at rest documentation

### PR DESCRIPTION
**Pull request by BlackYoup**
This adds a documentation about encryption at rest to replace the existing one we have here and there.

Let me know what you think about this.

### CODE REVIEW

File: administrate/encryption-at-rest.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+The encryption key is only stored encrypted on our systems and is not stored on your add-on. When your add-on boots, the key is downloaded over HTTPS from our systems and the disk is then mounted.
+
+We also have a small swap partition on add-ons to allow more available RAM for background services. This swap file is also encrypted to avoid any encrypted data leak.
+
+### Performances
+
+Performances may be a bit degraded with encryption enabled but in most of the cases under regular load, it is usually insignificant and an acceptable tradeoff.
+
+Also, depending on the fact that you are using a single vCPU add-on or a multiple vCPU one, numbers can greatly vary. Single vCPU instances will switch between decrypting / encrypting the data and reading / writing it to the disk while multiple vCPU instances will be able to parallelize those operations, leading to better performances.
```
Comment (ldoguin) [inline administrate/encryption-at-rest.md:33]:
```suggestion
Also, depending on wether you are using a single vCPU add-on or a multiple vCPU one, numbers can greatly vary. Single vCPU instances will switch between decrypting/encrypting the data and reading/writing it to the disk while multiple vCPU instances will be able to parallelize those operations, leading to better performances.
```

File: deploy/addon/mongodb.md [hunk 2]
```diff
@@ -89,8 +89,4 @@ The process consists in three steps:
 
 ## Encryption at rest
 
-MongoDB dedicated addons can be encrypted using LUKS with `aes-xts`.
-
-The passphrase is encrypted in our database using Cipher and Nonce as bytes arrays.
-
-To enable it, you need to ask to our support then we will perform invoicing configuration (more informations about pricing are available through support) and enable the encryption for your addon. Once it's done, you will need to migrate your addon then the encryption at rest will be up.
+Encryption at rest is available on MongoDB. You can have more information on the [dedicated page](/administrate/encryption-at-rest/)
```
Comment (ldoguin) [inline deploy/addon/mongodb.md:92]: To reference local document, please use hugo shortcode like https://www.clever-cloud.com/doc/contribute/shortcodes/#ref-shortcode

This allow us to make sure the link works even when using pathbegin CC magic like we do right now.

```suggestion
Encryption at rest is available on MongoDB. You can have more information on the [dedicated page]({{< ref "administrate/encryption-at-rest/" >}})
```

File: deploy/addon/mysql.md [hunk 3]
```diff
@@ -305,8 +305,4 @@ Keep in mind that usage of direct access is a trade-off: when you migrate your a
 
 ## Encryption at rest
 
-MySQL dedicated addons can be encrypted using LUKS with `aes-xts`.
-
-The passphrase is encrypted in our database using Cipher and Nonce as bytes arrays.
-
-To enable it, you need to ask to our support then we will perform invoicing configuration (more informations about pricing are available through support) and enable the encryption for your addon. Once it's done, you will need to migrate your addon then the encryption at rest will be up.
+Encryption at rest is available on MySQL. You can have more information on the [dedicated page](/administrate/encryption-at-rest/)
```
Comment (ldoguin) [inline deploy/addon/mysql.md:308]:
```suggestion
Encryption at rest is available on MySQL. You can have more information on the [dedicated page]({{< ref "administrate/encryption-at-rest/" >}})
```

File: deploy/addon/postgresql.md [hunk 4]
```diff
@@ -387,11 +387,7 @@ Every PostgreSQL database mananged by Clever Cloud comes with the following defa
 
 ## Encryption at rest
 
-PostgreSQL dedicated addons can be encrypted using LUKS with `aes-xts`.
-
-The passphrase is encrypted in our database using Cipher and Nonce as bytes arrays.
-
-To enable it, you need to ask to our support then we will perform invoicing configuration (more informations about pricing are available through support) and enable the encryption for your addon. Once it's done, you will need to migrate your addon then the encryption at rest will be up.
+Encryption at rest is available on PostgreSQL. You can have more information on the [dedicated page](/administrate/encryption-at-rest/)
```
Comment (ldoguin) [inline deploy/addon/postgresql.md:390]:
```suggestion
Encryption at rest is available on PostgreSQL. You can have more information on the [dedicated page]({{< ref "administrate/encryption-at-rest/" >}})
```

**Review by ldoguin — COMMENTED** [review]
:pray:

File: deploy/addon/mongodb.md (hunk 2 above)
Comment (BlackYoup) [reply deploy/addon/mongodb.md:92]: Oh okay, my bad

File: administrate/encryption-at-rest.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+- encryption
+- rest
+- security
+---
+
+## Introduction
+
+Encrypting your data at rest provides another security layer to protect your data from various security threats. By default and unless you implemented it in your code, the data stored on disk is not encrypted. Enabling encryption at rest allows you to make sure your data can't be read by anyone having a physical access to the hard drive.
+
+Encryption at rest only protects the data "at rest", meaning that if your data is sent over an unencrypted channel on the network, it could be read on the fly. The scenario encryption at rest covers are mostly hard drives stealing or unauthorized access once the hard drive got removed.
```
[the remark this answers is not in the document]
Comment (BlackYoup) [reply administrate/encryption-at-rest.md:13]: Eh, antidote told me not to use a `s` here so I dropped it, even though I wasn't really ok with this change either.

File: administrate/encryption-at-rest.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+The encryption key is only stored encrypted on our systems and is not stored on your add-on. When your add-on boots, the key is downloaded over HTTPS from our systems and the disk is then mounted.
+
+We also have a small swap partition on add-ons to allow more available RAM for background services. This swap file is also encrypted to avoid any encrypted data leak.
+
+### Performances
+
+Performances may be a bit degraded with encryption enabled but in most of the cases under regular load, it is usually insignificant and an acceptable tradeoff.
+
+Also, depending on wether you are using a single vCPU add-on or a multiple vCPU one, numbers can greatly vary. Single vCPU instances will switch between decrypting/encrypting the data and reading/writing it to the disk while multiple vCPU instances will be able to parallelize those operations, leading to better performances.
```
Comment (Geal) [inline administrate/encryption-at-rest.md:33]: whether

Comment (Geal) [reply administrate/encryption-at-rest.md:33]: better performance

**Pull request merged**